### ISSUE TITLE
cssnano with option --sourcemap does not save output to file but prints out in cli

### ISSUE THREAD

**Issue by feedmypixel** [ben-eb/cssnano#94]
# Source map Issue

#### Temp fix
I have found a temp fix which gives me sourcemaps this may be of help as a pointer
```
cssnano --sourcemap application.css application.css > application.min.css
```

---------------------------

If I run cssnano `3.4.0` using the `--sourcemap` flag via the cli I get the css file with sourcemap outputing in the cli but this does not save to the actual application.css file.
If I copy the output shown in the cli into my css file this works correctly and sourcemaps work as expected.

#### Of Note
- I am running the below call to cssnano via a bash script via npm scripts
- I have also tried this directly in the cli
- Without `--sourcemap` cssnano does whats expected

#### Code Example
```bash
cssnano -s application.css application.css
```
or
```bash
cssnano --sourcemap application.css application.css
```

#### Example
![cssnano](https://cloud.githubusercontent.com/assets/2305016/12054942/f0876760-af1f-11e5-8a14-455bbe8befd5.gif)

**Comment by ben-eb** [comment]
Thorough reporting, many thanks. Bonus points for gifs. :+1:

Released in cli v1.0.2.

**Comment by feedmypixel** [comment]
Life is so much easier with gifs :)

To confirm all working here :+1:
